docs/components/modeler/forms/configuration/forms-config-templating-syntax.md

```diff
@@ -0,0 +1,136 @@
+---
+id: forms-config-templating-syntax
+title: Templating syntax
+description: Learn about templated properties configuration, which provides dynamic content creation within forms using a templating language called feelers.
+---
+
+Templated properties configuration allows for dynamic content creation within forms using a templating language called **feelers**.
+
+## Feelers syntax
+
+### Variables/inserts
+
+To insert a variable, use double curly braces `{{variable}}`, and the value of this variable will be inserted. You can use **any valid feel expression** within these double braces.
+
+```
+Hello {{username}}, you are {{if isAdmin then "an admin" else "a user"}}.
+```
+
+### Iterating through arrays
+
+Iterate through arrays using the _loop_ tags. Within the loop, reference each array element with `{{this}}`, or if your array elements are objects, via their properties. To access data outside the scope of the individual items, use the `{{parent}}` accessor.
+
+Working with this list of items, with a currency symbol defined outside of the list may be achieved so.
+
+**Data**
+
+```json
+{
+  "currency": "$",
+  "items": [
+    {
+      "name": "bananas",
+      "price": 2.5
+    },
+    {
+      "name": "mangos",
+      "price": 4
+    },
+    {
+      "name": "strawberries",
+      "price": 3
+    }
+  ]
+}
+```
+
+**Template**
+
+```
+{{#loop items}}
+Item name: {{name}}
+Item price: {{parent.currency}}{{price}}
```

Comment:
Should we use indentation here or is this unusual for feelers?

Reply:
Indentation would render the spacing. It's a downside but it's one that's common across text-templating languages in general, mustache, handlebars, ect